src/inputrules.ts

```diff
@@ -11,6 +11,7 @@ export class InputRule {
 
   /// @internal
   undoable: boolean
+  inCode?: 'allowed' | 'only';
```

Comment:
Could you make this `boolean | "only"`, so people can also explicitly provide `false`?

Reply:
done.